anchors/anchor_repository_integration_test.go

```diff
@@ -31,13 +33,68 @@ func TestMain(m *testing.M) {
 	os.Exit(result)
 }
 
-func TestCommitAnchor_Integration(t *testing.T) {
+func TestPreCommitAnchor_Integration(t *testing.T) {
 	anchorID := utils.RandomSlice(32)
+	signingRoot := utils.RandomSlice(32)
+
+	anchorIDTyped, _ := anchors.ToAnchorID(anchorID)
```

Comment:
lets not ignore the errors here. If at some point ToAnhorID logic changes, this will fail silently. And that is a debugging nightmare for that person who is making the change